Arrowgene.Ddon.GameServer/Party/PartyGroup.cs

```diff
@@ -714,6 +714,11 @@ private int TakeSlot(PartyMember partyMember)
                     }
                 }
 
+                if (slotIndex == InvalidSlotIndex)
+                {
+                    return slotIndex;
+                }
```

Comment:
What does this imply exactly?

Reply:
I'm not too sure that we'd want to return an invalid value instead of throwing an exception and handling it from the callers, even if we avoid the out of range exception here, the value -1 is invalid and would cause more issues later on, wouldn't it? this would just silence them in their origin